README

```diff
@@ -108,6 +108,10 @@ Mandriva:
 	Mandriva has integrated fio into their package repository, so installing
 	on that distro should be as easy as typing ``urpmi fio``.
 
+Arch Linux:
+        Arch Linux package is provided under community.
```

Comment:
I've provided an update with your feedback, I think its a good trade-off for anyone who is not familiar with Arch too.

Indeed, lot of thing go really quick, the maintainer do a good job!